src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx

```diff
@@ -19,13 +20,77 @@ type HacbsApplicationDetailsProps = {
 const HacbsApplicationDetails: React.FC<HacbsApplicationDetailsProps> = ({ applicationName }) => {
   const { namespace } = React.useContext(NamespaceContext);
   const navigate = useNavigate();
+  const { quickStarts } = useChrome();
+
   const [application, applicationsLoaded] = useK8sWatchResource<ApplicationKind>({
     groupVersionKind: ApplicationGroupVersionKind,
     name: applicationName,
     namespace,
   });
   const appDisplayName = application?.spec?.displayName || '';
 
+  useEffect(() => {
+    const hacbsShowAppQs = localStorage.getItem('hacbsShowAppQs');
```

Comment:
Let's use the convention `hacbs/` as a prefix for all localstorage properties related to hacbs.

Reply:
implemented